src/main/java/frc/robot/subsystems/drive/PhoenixDrive.java

```diff
@@ -139,6 +190,31 @@ public void setGoalSpeeds(ChassisSpeeds goalSpeeds, boolean fieldCentric) {
         this.setControl(request);
     }
 
+    // SYS ID
+
+    public void setSysIdRoutine(SysIdRoutineType routineType) {
+        switch (routineType) {
```

Comment:
This should have a default case (even if there is nothing to do by default). Also, each of these cases needs a `break` statement to avoid falling through to the next state.

Reply:
got it